packages/electron-commons/src/initializers/electron-node.ts

```diff
@@ -96,8 +112,16 @@ export const initializeNodeEnvironment: EnvironmentInitializer<
 
     return {
         id: env.env,
-        // eslint-disable-next-line @typescript-eslint/no-misused-promises
         dispose: async () => {
+            const envDisposed = executeRemoteCall<NodeEnvironmentDisposeCommand, NodeEnvironmentDisposedEvent>(
+                child,
+                { id: 'nodeEnvironmentDisposeCommand' },
+                'nodeEnvironmentDisposedEvent'
+            );
+
+            // wait for dispose finished or timeout
+            await Promise.allSettled([timeout(envDisposed, 1_000)]);
```

Comment:
1s timeout? why?

Reply:
to terminate process if dispose is taking a long time